### ISSUE TITLE
Website request: Support permalinks to specific backends in (Reference) Gallery

### ISSUE THREAD

**Issue by jonmmease** [holoviz/holoviews#4619]
#### Describe the solution you'd like

I'd like to be able to reference the HoloViews Gallery and Reference Gallery for a specific backend (Plotly in my case).  

Right now the URLs for the *Gallery* (http://holoviews.org/gallery/) and *Reference Gallery* (http://holoviews.org/reference/) pages always default to showing the Bokeh backend. You can switch to the Matplotlib and Plotly backends using the buttons at the top, but this doesn't change the URL path so it doesn't look like there is currently a way to use a URL to land and the page for the Matplotlib or Plotly backends.

I'm not familiar with how the documentation is structured, but would there be a way to assign a separate subpath or anchor to the different backends?

I'd like to be able to reference the Plotly elements from some future Dash documentation.

**Comment by jbednar** [comment]
Good question. Each individual Plotly element can be referenced already:

http://holoviews.org/reference/elements/plotly/Bars.html
http://holoviews.org/reference/elements/plotly/Ellipse.html

But I don't see a way to refer to the Plotly index page, which I think there should be.

**Comment by jlstevens** [comment]
I agree this is a good idea and could probably be addressed with a little bit of javascript and a url query parameter (to avoid having to restructure all the HTML).
